examples/config/networks/zksync_era_sepolia.json

```diff
@@ -0,0 +1,17 @@
+{
+  "network_type": "EVM",
+  "slug": "zksync_era_sepolia",
+  "name": "ZkSync Era Sepolia",
+  "rpc_urls": [
+    {
+      "type_": "rpc",
+      "url": "https://sepolia.era.zksync.dev",
+      "weight": 100
+    }
+  ],
+  "chain_id": 300,
+  "block_time_ms": 5000,
+  "confirmation_blocks": 12,
```

Comment:
Not a blocker, but if there's an **easy** way to find out what block confirmation times these networks have, it's worth adjusting it.

12 is for Ethereum, others might have shorter block finalisation.